index.html

```diff
@@ -1,122 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Is it ok to delete this file?

Reply:
It contained unfinished code so I assumed so - app runs fine without this file.